scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -220,7 +220,7 @@
 	language	Khmer		79	khm	km
 	language	Kikuyu, Gikuyu		80	kik	ki
 	language	Kinyarwanda		81	kin	rw
-	language	Kyrgyz		82											
+	language	Kirghiz, Kyrgyz		82	kir	ky									
```

Comment:
I'm almost certain that this is not what we want to do. If the goal is to have both the "Kirghiz" and "Kyrgyz" spellings to be accepted as valid. Because the above would mean that either is invalid by itself, and only the full literal string "Kirghiz, Kyrgyz" is acceptable. So we should make the other spelling an alternate here as well, as in:
`language	Kyrgyz		82	kir	ky	Kirghiz`

Just as I typed this, I realized that we have quite a few of such comma-separated entries in the block already!! `Navajo, Navaho` etc. 

We will need to fix them all. And there is no way to do that, other than via a database update.

Reply:
(To correct myself, there is definitely a way to address this without a Flyway update - for example, we can keep `Navajo, Navaho` as the main name, but add each of the 2 formats as a separate alternate, as in: 
`language	Navajo, Navaho		109	nav	nv	Navajo	Navaho`)